domain-services/ifix-master-data-service/src/main/java/org/egov/repository/ChartOfAccountRepository.java

```diff
@@ -38,26 +38,8 @@ public void save(ChartOfAccount chartOfAccount) {
         return (mongoTemplate.find(searchQuery,ChartOfAccount.class));
     }
 
-    /**
-     *
-     * @param url
-     * @param govtSearchRequest
-     * @return
-     */
-    public List<Government> searchTenants(String url, GovernmentSearchRequest govtSearchRequest) {
-
-        GovernmentResponse governmentResponse = null;
-        try {
-            governmentResponse = restTemplate.postForObject(url, govtSearchRequest, GovernmentResponse.class);
-        } catch (HttpClientErrorException e) {
-            log.error("Government Service threw an Exception: ", e);
-            throw new ServiceCallException(e.getResponseBodyAsString());
-        } catch (Exception e) {
-            log.error("Exception while fetching from searcher: ", e);
-        }
-
-        if (governmentResponse != null)
-            return governmentResponse.getGovernment();
-        return Collections.emptyList();
+    public List<ChartOfAccount> searchCoaCode(ChartOfAccount chartOfAccount) {
```

Comment:
Can't we keep this part of the normal search method instead of making this separate method?

Reply:
done..(This one we are not using hence removed).